src/baseVisitor.ts

```diff
@@ -90,199 +89,346 @@ export class BaseVisitor<T> implements Required<RecursiveVisitors<T>> {
       cb(n.body, st)
     }
   }
-  ClassDeclaration<S>(_n: swc.ClassDeclaration, _st: S, _cb: Callback<S>) {
-    // not implemented
-  }
-  ClassExpression<S>(_n: swc.ClassExpression, _st: S, _cb: Callback<S>) {
-    // not implemented
-  }
-  ClassMember<S>(_n: swc.ClassMember, _st: S, _cb: Callback<S>) {
-    // not implemented
-  }
-  ClassMethod<S>(_n: swc.ClassMethod, _st: S, _cb: Callback<S>) {
-    // not implemented
-  }
-  ClassProperty<S>(_n: swc.ClassProperty, _st: S, _cb: Callback<S>) {
-    // not implemented
-  }
-  Computed<S>(_n: swc.ComputedPropName, _st: S, _cb: Callback<S>) {
-    // not implemented
-  }
-  ConditionalExpression<S>(_n: swc.ConditionalExpression, _st: S, _cb: Callback<S>) {
-    // not implemented
+  ClassDeclaration<S>(n: swc.ClassDeclaration, st: S, cb: Callback<S>) {
+    for (const decorator of n.decorators ?? []) {
+      cb(decorator, st)
+    }
+    cb(n.identifier, st)
```

Comment:
This is simply a style preference of mine, but you can fix all of those `Expected blank line before this statement` lint errors by running `npm run lint:fix`.